go/client/cmd_simplefs.go

```diff
@@ -44,14 +44,17 @@ func NewCmdSimpleFS(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Comm
 	}
 }
 
+const mountDir = "/keybase"
+
 func makeSimpleFSPath(g *libkb.GlobalContext, path string) keybase1.Path {
-	mountDir := "/keybase"
 
 	path = filepath.ToSlash(path)
 	if strings.HasSuffix(path, "/") {
 		path = path[:len(path)-1]
 	}
 
+	// Test for the special mount dir prefix before the absolute test.
+	// Otherwise the current dir will be prepended, below.
```

Comment:
If `mountDir` is a prefix of `path`, though, doesn't that mean `path` has to be absolute? Shouldn't the current-dir-prepending on line 65 get skipped in that case?